README.md

```diff
@@ -53,7 +53,7 @@ The paper and other materials on COSMA are available under the following link:
 - **Custom Data Layout Support:** natively uses its own blocked data layout of matrices, but supports arbitrary grid-like data layout of matrices.
 - **Tranposition/Conjugation Support:** matrices `A` and `B` can be transposed and/or conjugated.
 - **Communication and Computation Overlap:** supports overlapping of communication and computation.
-- **Spack Installation:** can be built and installed with `Spack`.
+- **Spack Installation:** can be built and installed with `Spack` since v14.1
 
 ## Building COSMA
 
```

Comment:
Since github does not allow me to comment on a part of code that has not been changed by the PR, let me write my suggestions here:
- it would be nice to add few lines on how to install COSMA using Spack in the `INSTALL` file.
- if you think it would be a good fit, you can also add a separate section/paragraph in the `README` on Spack installation.
- In `README`, in `Using COSMA in 30 seconds` section, maybe you can add a line after the github example, like: "Alternatively, if `Spack` is available, you can instead do...".

These are just the suggestions, so feel free to do however you prefer :)